checks/evaluation/security_policy.go

```diff
@@ -19,6 +19,81 @@ import (
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+func scoreSecurityCriteria(f checker.File, info []checker.SecurityPolicyInformation, dl checker.DetailLogger) int {
+	var urls, emails, discvuls, linkedContentLen, score int
+	// for Security-Policy, EndOffset denotes the
```

Comment:
`EndOffset` is typically used to identify the end of the relevant file where a Snippet value was found. In your case, Snippet would be an email. address, for example.

Reply:
I agree - my "overload" using `EndOffset` is not well polished. for my proposed security check improvement, I want to ensure that there is some text beyond simply emails and urls (linked content). To do this, I'd like to know just how much content is in the found policy file at the time of scoring.

I have two options as I see it (perhaps there is a better option)

1. add a new attribute/field/member to the struct `File` - something like (`FileLength`) but that would break the purpose used for code snippets--so likely not a good option.
2. (my choice) add an explicit attribute to the `SecurityPolicyData` struct which conveys that files content size

To that end, the new `SecurityPolicyData` struct appears as:
```
type SecurityPolicyData struct {
        // security policy information found in repo or org
        Information []SecurityPolicyInformation
        // total size in bytes of the security policy file contents
        SecurityContentLength uint
        // file that contains the security policy information
        // only looking for one file
        File File
}
```